client/app/utils/places.js

```diff
@@ -0,0 +1,151 @@
+/*
+ Generic utilities for dealing with Google Maps place objects.
+*/
+
+const toRadians = (degrees) => degrees * (Math.PI / 180); // eslint-disable-line no-magic-numbers
+
+/**
+ * Calculate the distance between two points.
+ *
+ * @param {google.maps.LatLng} pointA - first point
+ * @param {google.maps.LatLng} pointB - second point
+ *
+ * @return {Number} distance between the given points (in km?)
+ */
+const computeScale = (pointA, pointB) => {
```

Comment:
I think radius isn't a correct term, and in any case "distance" is exactly what we're calculating here. The change is very minor, I can do it myself as well :)